training/model_evaluation/components/compute_metrics/spec.yaml

```diff
@@ -0,0 +1,84 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: compute_metrics
+display_name: Compute Metrics
+description: Calculate model performance metics, given ground truth and prediction data.
+
+version: 0.0.1.preview
+type: command
+
+tags:
+  type: evaluation
+  sub_type: compute_metrics
+  Preview: ""
+
+inputs:
+  task:
+    type: string
+    default: tabular-classification
+    enum: [tabular-classification, 
+          tabular-classification-multilabel, 
+          tabular-regression, 
+          text-classification,
+          text-classification-multilabel,
+          text-named-entity-recognition, 
+          text-summarization, 
+          question-answering, 
+          text-translation]
+    description: "Task type"
+  ground_truth:
+    type: uri_file
+    optional: true
+    description: "Ground Truths of Test Data as a 1-column JSON Lines file"
+  ground_truth_mltable:
+    type: mltable
+    optional: true
+    description: "Ground Truth of Test Data as MLTable folder"
+  ground_truth_column_name:
+    type: string
+    optional: true
+    description: "Column name which contains ground truths in provided uri file for ground_truth. (Optional if we have one column name.)"
+  prediction:
+    type: uri_file
+    optional: true
+    description: "Model Predictions as a 1-column JSON Lines file"
+  prediction_mltable:
+    type: mltable
+    optional: true
+    description: "Model Predictions as MLTable folder"
+  prediction_column_name:
+    type: string
+    optional: true
+    description: "Column name which contains ground truths in provided uri file for prediction. (Optional if we have one column name.)"
+  prediction_probabilities:
+    type: uri_file 
+    optional: true
+    description: "Predictions Probabilities as 1-column JSON Lines file"
+  prediction_probabilities_mltable:
+    type: mltable
+    optional: true
+    description: "Predictions Probabilities as MLTable folder"
+  evaluation_config:
+    type: uri_file 
+    optional: true
+    description: "Additional parameters required for evaluation."
+
+outputs:
+  evaluation_result:
+    type: uri_folder
+
+code: ../../src
+environment: azureml:model-evaluation:0.0.1-preview
```

Comment:
this should have integer version

Reply:
We have kept the version  as 0.0.1-preview in Env asset. https://github.com/Azure/azureml-assets/blob/c1b563e1b3799ac5ca6215ebf5582c96b71f7618/training/model_evaluation/environment/asset.yaml#L2

Reply:
Shall I make it to auto?

Reply:
version: auto is set in the asset.yaml file: https://github.com/Azure/azureml-assets/wiki/Environments#assetyaml

Reply:
Updated